src/current/cockroachcloud/export-metrics.md

```diff
@@ -338,6 +338,67 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
 Exporting Metrics to Azure Monitor from a CockroachDB {{ site.data.products.dedicated }} cluster is in **[limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability)** and is only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. This feature is subject to change.
 {{site.data.alerts.end}}
 
+Exporting metrics to Azure Monitor is only available on CockroachDB {{ site.data.products.dedicated }} clusters that are hosted on Azure. If your CockroachDB {{ site.data.products.dedicated }} cluster is hosted on AWS, you can export metrics to [AWS CloudWatch]({% link cockroachcloud/export-metrics.md %}?filters=aws-metrics-export). If your CockroachDB {{ site.data.products.dedicated }} cluster is hosted on GCP, you can export metrics to [Datadog]({% link cockroachcloud/export-metrics.md %}?filters=datadog-metrics-export) or [Prometheus]({% link cockroachcloud/export-metrics.md %}?filters=prometheus-metrics-export) instead.
+
+Perform the following steps to enable metrics export from your CockroachDB {{ site.data.products.dedicated }} cluster to Azure Monitor.
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID that will be used in Step 5:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. The ID should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+
+1. Create a private key and certificate pair, using a tool such as OpenSSL. Refer to [this example]({% link {{site.current_cloud_version}}/create-security-certificates-openssl.md %}#step-1-create-the-ca-key-and-certificate-pair) to create a private key file (`ca.key`) and a certificate file (`ca.crt`) that will be uploaded in Step 3.
```

Comment:
```suggestion
1. Create a private key and certificate pair, using a tool such as OpenSSL. Refer to [Create the CA key and certificate pair]({% link {{site.current_cloud_version}}/create-security-certificates-openssl.md %}#step-1-create-the-ca-key-and-certificate-pair). This pair will be uploaded in step 3. Name the private key file (`ca.key`) and name the certificate file (`ca.crt`).
```

Reply:
Updated step, but not as suggested because the suggestion is not correct. Only the certificate file is uploaded in step 3.